src/ensembl/src/content/app/browser/interstitial/browser-interstitial-instructions/BrowserInterstitialInstructions.tsx

```diff
@@ -0,0 +1,96 @@
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { push } from 'connected-react-router';
+
+import * as urlFor from 'src/shared/helpers/urlHelper';
+
+import { Step } from 'src/shared/components/step/Step';
+import { ImageButton } from 'src/shared/components/image-button/ImageButton';
+import { PrimaryButton } from 'src/shared/components/button/Button';
+import { Status } from 'src/shared/types/status';
+
+import { ReactComponent as SpeciesSelectorIcon } from 'static/img/launchbar/species-selector.svg';
+import { ReactComponent as BrowserIcon } from 'static/img/launchbar/browser.svg';
+import { ReactComponent as SearchIcon } from 'static/img/sidebar/search.svg';
+
+import styles from './BrowserInterstitialInstructions.scss';
+
+const BrowserInterstitialInstructions = () => {
+  const dispatch = useDispatch();
+
+  const goToSpeciesSelector = () => {
+    const url = urlFor.speciesSelector();
+    dispatch(push(url));
+  };
+
+  return (
+    <section className={styles.instructionsPanel}>
+      <div className={styles.instructionsWrapper}>
+        <div className={styles.stepWrapper}>
+          <Step count={1} title="Find and add a species" />
+          <div className={styles.description}>
+            <ImageButton
+              className={styles.imageButtonIcon}
+              status={Status.DISABLED}
+              image={SpeciesSelectorIcon}
+            />
+            <div className={styles.iconLabel}>Species Selector</div>
+          </div>
+        </div>
+
+        <div className={styles.stepWrapper}>
+          <Step count={2} title="Return to this app" />
+          <div className={styles.description}>
+            <ImageButton
+              className={styles.imageButtonIcon}
+              status={Status.DISABLED}
+              image={BrowserIcon}
+            />
+            <div className={styles.iconLabel}>Genome Browser</div>
+          </div>
+        </div>
+
+        <div className={styles.stepWrapper}>
+          <Step
+            count={3}
+            title="Use Search or the example links to view a gene or region"
+          />
+          <div className={styles.searchDescription}>
+            <ImageButton
+              className={styles.searchButtonIcon}
+              status={Status.DISABLED}
+              image={SearchIcon}
+            />
+            <div className={styles.exampleText}>Example gene</div>
+            <div className={styles.exampleText}>Example region</div>
+          </div>
+        </div>
+
+        <PrimaryButton
```

Comment:
Didnt we decide to use Wrap click buttons around a Link so that we can right-click and open in new tab?

Reply:
Did we? I forgot... If we did, we should style the link so that it looks like a button; we shouldn't wrap a button with a link.

Reply:
After another thought, I suggest we separate this task from this PR as well. It goes deeper than just replacing a button with a link here. If we want true links that look like buttons, then we should either create a dedicated component for that or update the existing PrimaryButton to handle this case. Is still feels wrong to me, because I would not expect people to interact with a button in the way they interact with a link (i.e. try to right-click on it and 'open' it in another window)